internal/daysteps/daysteps.go

```diff
@@ -1,20 +1,61 @@
 package daysteps
 
 import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/spentcalories"
 )
 
 const (
-	// Длина одного шага в метрах
-	stepLength = 0.65
-	// Количество метров в одном километре
-	mInKm = 1000
+	stepLength = 0.7
+	mInKm      = 1000
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	parts := strings.Split(data, ",")
+	if len(parts) != 2 {
+		return 0, 0, fmt.Errorf("некорректный формат строки: %s", data)
+	}
+
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil || steps <= 0 {
+		return 0, 0, fmt.Errorf("ошибка преобразования шагов: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return 0, 0, fmt.Errorf("ошибка преобразования шагов: %w", err)
```